README.md

```diff
@@ -19,7 +19,8 @@ OpenEBS holds a monthly community meeting via Zoom on the last Thursday of the m
 <br>
 The next meeting is on: `Thursday 31 October, at 14:00 UTC`
 <br>
-Meeting Link: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1
+Meeting Link: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1  
+Calendar Link: https://tockify.com/cncf.public.events/detail/837/1738245600000
```

Comment:
But this is a single event, it's not a series?

Reply:
There's a .ics file on the tockify page. I'd agree with you on the part that the .ics file isn't readily apparent.